load.php

```diff
@@ -29,6 +29,7 @@
 if ( ! defined( 'PERFLAB_OBJECT_CACHE_DROPIN_VERSION' ) ) {
 	define( 'PERFLAB_OBJECT_CACHE_DROPIN_VERSION', false );
 }
+define( 'PERFLAB_OBJECT_CACHE_DROPIN_LATEST_VERSION', 2 );
```

Comment:
Let's add a unit test that verifies `PERFLAB_OBJECT_CACHE_DROPIN_LATEST_VERSION` is identical to the value in `server-timing/object-cache.copy.php`

Reply:
Added in https://github.com/WordPress/performance/pull/1047/commits/56ec805a7925c6223394a547fba8c2e28b8b91e7